docs/docs/03-tools/04-credential-tools.md

```diff
@@ -166,41 +162,24 @@ This can be overridden with a credential alias, i.e. `credential: my-cred-tool.g
 If the credential has an alias, use it instead of the tool name when you specify an override.
 :::
 
-The `--credential-override` argument must be formatted in one of the following three ways:
+The `--credential-override` argument must be formatted in one of the following two ways:
 
 #### 1. Key-Value Pairs
 
-`toolA:ENV_VAR_1=value1,ENV_VAR_2=value2;toolB:ENV_VAR_1=value3,ENV_VAR_2=value4`
+`toolA:ENV_VAR_1=value1,ENV_VAR_2=value2`
+
+In this example, both `toolA` provides the variables `ENV_VAR_1` and `ENV_VAR_2`.
+This will set the environment variables `ENV_VAR_1` and `ENV_VAR_2` to the specific values `value1` and `value2`.
 
-In this example, both `toolA` and `toolB` provide the variables `ENV_VAR_1` and `ENV_VAR_2`.
-This will set the environment variables `ENV_VAR_1` and `ENV_VAR_2` to the specific values provided for each tool.
+:::info
+To override more than one credential, use `;` as a separator. For example, `toolA:ENV_VAR_1=value1;toolB:ENV_VAR_2=value2`.
+:::
 
 #### 2. Environment Variables
 
```

Comment:
remove `toolB` and its env vars here to match the updated example

Reply:
Thanks for catching this. Just pushed the fix